src/clr-core/styles/utils/_close.scss

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+// scss-lint:disable QualifyingElement
+button.close {
```

Comment:
Why is our preference here to create global CSS styles over either reusing/extending another component or making a component like `cds-close-button` that we can re-use?

This seems inconsistent with how we've approached things thus far.

Reply:
I think creating a reusable `cds-close-button` makes sense here. 👍 Will work on this after the latest button PR (with icon button) is merged!

Reply:
Pulling in CSS into a component style sheet is not shared its duplicated across every component because of how sass compiles out the CSS. Because of this we should avoid patterns like this if we can.

Best bet would be to get `<cds-icon-button>` in and see if that meets our needs for a close button.
However we could get internal components up and running. I outlined what would need to be done. Since its non trivial amount of work it could be done as a follow up PR to this one. https://github.com/vmware/clarity/issues/4501

Reply:
Resolving so that this is addressed separately after this PR is merged.